demisto_sdk/commands/validate/validators/DO_validators/DO106_docker_image_is_not_native_image.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Script]
+
+
+class DockerImageIsNotNativeImageValidator(BaseValidator[ContentTypes]):
+    error_code = "DO106"
```

Comment:
Let's keep DO106 as the "not the latest docker validation"

Reply:
done